lib/ui/screens/contact_list_screen.dart

```diff
@@ -134,7 +120,8 @@ class _ContactListScreenState extends State<ContactListScreen> {
                             else return Card(child: Text("Empty Contact"));
                           }
                       );
-                    return nullBox();
+                    return NullScreen(message: "You haven't had any contacts yet",
```

Comment:
Should not use nested Scaffold (Scaffold inside another Scaffold). 
Remove NullScreen class, instead, build a widget here for empty message is enough